home-screen-widget/profiles/components/ProfileCell.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <ClickablePane @click="onClick">
+        <div class="profile-cell">
+            <div class="profile-cell__cover">
+                <img
+                    v-if="profile.avatar"
+                    :src="profile.avatar"
+                    :alt="profile.firstname"
+                >
+            </div>
+            <div class="profile-cell__name">
+                {{ profile.firstname ? profile.firstname : profile.name }}
```

Comment:
Why not just use the `display_name`?

Reply:
I had the display name initially but since it was longer I thought it would get ugly if it gets clipped.  I can add it back though.